src/sql/workbench/contrib/notebook/browser/cellViews/codeCell.component.ts

```diff
@@ -110,4 +126,20 @@ export class CodeCellComponent extends CellView implements OnInit, OnChanges {
 	get isStdInVisible(): boolean {
 		return this.cellModel.stdInVisible;
 	}
+
+	public getEditor(): any {
```

Comment:
any | undefined. Shouldn't this be BaseTextEditor anyways as the type?

Reply:
yea but when we know for sure there's no editor for the particular cell, why add it and return undefined.

Reply:
The whole point of typescript is to make it so we don't have unknown behavior like this. Always be as specific as you can. (this will matter once we enable strict null checking - but for now is just good practice)